e2e/env/regressions.go

```diff
@@ -126,13 +126,13 @@ func (c ctx) issue43(t *testing.T) {
 	)
 }
 
-// https://github.com/sylabs/singularity/issues/274
+// https://github.com/sylabs/apptainer/issues/274
 // The conda profile.d script must be able to be source'd from %environment.
 // This has been broken by changes to mvdan.cc/sh interacting badly with our
 // custom internalExecHandler.
 // The test is quite heavyweight, but is warranted IMHO to ensure that conda
 // environment activation works as expected, as this is a common use-case
-// for SingularityCE.
+// for ApptainerCE.
```

Comment:
Change from ApptainerCE to Apptainer; the original must have been a mistake made during porting something from sylabs.

Reply:
of course you are correct, and these can be resolved by later Issue/PR rather than blocking this one

This is not a case where Athena needs to leap from Jupiter's head fully formed

Reply:
submitting update